test_scripts/API/VehicleData/GetVehicleData/005_GetVD_versioning.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TO ADD !!!
+-- Use case: TO ADD !!!
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetVehicleData] As a mobile app wants to send a request to get the details of the vehicle data
+--
+-- Description:
+-- In case:
+-- mobile application sends valid GetVehicleData to SDL and this request is allowed by Policies
+-- SDL must:
+-- 1) Transfer this request to HMI
+-- 2) After successful response from hmi
+--    respond SUCCESS, success:true and parameter value received from HMI to mobile application
+---------------------------------------------------------------------------------------------------
```

Comment:
@YarikMamykin Please update description in a way it has:
1) all required sections completed properly
2) not required/not applicable sections removed
3) mention about versioning (the case this script checks)

Reply:
@dboltovskyi , fixed in 0c3f6df4ce34a698429a81f8a386c026c5389490